ibmjava/content.md

```diff
@@ -0,0 +1,99 @@
+-	[s390x](https://hub.docker.com/r/s390x/ibmjava/)
+
+### How to use this Image
+
+To run a pre-built jar file with the JRE image, use the following commands:
+
+```dockerfile
+FROM ibmjava:jre
+RUN mkdir /opt/app
+COPY japp.jar /opt/app
+CMD ["java", "-jar", "/opt/app/japp.jar"]
+```
+
+You can build and run the Docker Image as shown in the following example:
+
+```console
+docker build -t japp .
+docker run -it --rm japp
+```
+
+If you want to place the jar file on the host file system instead of inside the container, you can mount the host path onto the container by using the following commands:
+
+```dockerfile
+FROM ibmjava:jre
+CMD ["java", "-jar", "/opt/app/japp.jar"]
+```
+
+```console
+docker build -t japp .
+docker run -it -v /path/on/host/system/jars:/opt/app japp
+```
+
+### Using the Class Data Sharing feature
+
+IBM SDK, Java Technology Edition provides a feature called [Class data sharing](http://www-01.ibm.com/support/knowledgecenter/SSYKE2_8.0.0/com.ibm.java.lnx.80.doc/diag/understanding/shared_classes.html). This mechanism offers transparent and dynamic sharing of data between multiple Java virtual machines (JVMs) running on the same host thereby reducing the amount of physical memory consumed by each JVM instance. By providing partially verified classes and possibly pre-loaded classes in memory, this mechanism also improves the start up time of the JVM.
+
+To enable class data sharing between JVMs that are running in different containers on the same host, a common location must be shared between containers. This requirement can be satisfied through the host or a data volume container. When enabled, class data sharing creates a named "class cache", which is a memory-mapped file, at the common location. This feature is enabled by passing the `-Xshareclasses` option to the JVM as shown in the following Dockerfile example:
+
+```dockerfile
+FROM ibmjava:jre
+RUN mkdir /opt/shareclasses
+RUN mkdir /opt/app
+COPY japp.jar /opt/app
+CMD ["java", "-Xshareclasses:cacheDir=/opt/shareclasses", "-jar", "/opt/app/japp.jar"]
+```
+
+The `cacheDir` sub-option specifies the location of the class cache. For example `/opt/sharedclasses`. When sharing through the host, a host path must be mounted onto the container at the location the JVM expects to find the class cache. For example:
+
+```console
+docker build -t japp .
+docker run -it -v /path/on/host/shareclasses/dir:/opt/shareclasses japp
+```
+
+When sharing through a data volume container, create a named data volume container that shares a volume.
+
+```console
+docker create -v /opt/shareclasses --name classcache japp /bin/true
+```
+
+Then start your JVM container by using `--volumes-from` flag to mount the shared volume, as shown in the following example:
+
+```console
+docker run -it --volumes-from classcache japp
+```
+
+### See Also
+
+See the [Websphere-Liberty image](https://hub.docker.com/_/websphere-liberty/), which builds on top of this IBM docker image for Java.
```

Comment:
Woot, I was going to ask whether the plan was to update the `websphere-liberty` image to be `FROM ibmjava:...` :smile: